colcon_notification/desktop_notification/terminal_notifier.py

```diff
@@ -73,3 +79,20 @@ def notify(self, *, title, message, icon_path=None):  # noqa: D102
             cmd_str = ' '.join(cmd)
             logger.error(
                 "Failed to invoke '{cmd_str}'".format_map(locals()))
+
+
+def _get_prefix_path(path):
+    path = Path(path)
+    while path.name:
```

Comment:
Although it is unlikely, someone might have `lib` somewhere higher in their build path (build path since it'd be the symlink-install case that ends up traversing higher and higher), and since we indefinitely traverse parent directories we would find it. I thought there might be a chance we could use `package_data` + `pkg_resources.has_resource` but I [see from this comment](https://github.com/colcon/colcon-notification/blob/353b6b7afcd5eb3987657935d5ebc140230b9442/setup.py#L47) that that is not feasible.

We could go up at most 2 directories (since it's going to be e.g. `lib/python3.6/site-packages`) to make this less fragile, but I doubt someone would really run into this edge case anyway.

Reply:
Good point. I removed the loop in favor of checking the second level parent name in 6c8ea6f2128336f420406fffd052c0c63e11c3dd.